src/test/kotlin/com/github/jensim/megamanipulator/actions/vcs/githubcom/GithubComClientTest.kt

```diff
@@ -58,6 +62,32 @@ class GithubComClientTest {
         json = SerializationHolder.instance.readableJson
     )
 
+    @Test
```

Comment:
Maybe this test should should be part of a bigger flow. 
IntegrationTest like
Search
Clone
Apply changes
Commit & push (with lazy fork being created)
Draft PR
Reword PR
Decline PR
Remove Fork

Reply:
Agree